roles/lsaai/templates/pre_10_setup_connection

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Pre-file for LDAP service providing authz information.
+LDAP_LOGIN="{{ ldap_domains['stack']['base'] }}"
+LDAP_PASSWORD="{{ ldap_credentials['stack']['admin']['pw'] }}"
+LDAP_URL="ldaps://{% for network_id in ip_addresses[inventory_hostname] %}
+      {%- if ip_addresses[inventory_hostname][network_id]['fqdn'] is defined and
+             ip_addresses[inventory_hostname][network_id]['fqdn'] | length >= 1 and
+             ip_addresses[inventory_hostname][network_id]['fqdn'] != 'NXDOMAIN' -%}
+          {{ ip_addresses[inventory_hostname][network_id]['fqdn'] }}
+      {%- endif %}
+    {%- endfor %}:636"
```

Comment:
This for loop could result in multiple values concatenated or no value listed at all -> not working URL. The suggestion below:
 * Is not perfect either as it assumes there will be one and only one address with not empty neither null `fqdn` key for the machine, but if present and not `NXDOMAIN`, it will use the `fqdn` and otherwise it will use the IP address
 * Uses `#jinja2: trim_blocks:True, lstrip_blocks: True`, so you do not need to specify that for individual jinja tags.

Reply:
Yes I forgot to check that there is not more than one defined `fqdn`. While it would be the first machine of all our playbooks to have more than one `fqdn` defined, but true I should check against it. But I don't see the need for changing entire code when a two lines changed is also working.

Reply:
Check the updated code - unfortunately it does not appear here automatically